source/03-components/breadcrumb/breadcrumb.twig

```diff
@@ -5,28 +5,11 @@
   modifier_classes ? modifier_classes : '',
 ]|join(' ')|trim %}
 
-{% set title_classes = [
-  'c-breadcrumb__title',
-  title_modifier_classes ? title_modifier_classes : '',
-  hide_title ? 'u-visually-hidden' : '',
-]|join(' ')|trim %}
-
-{% set breadcrumb_id = 'breadcrumb-label'|clean_unique_id %}
-
 <nav {{ add_attributes({
-  'aria-labelledby': breadcrumb_id,
+  'aria-label': label|default('Breadcrumb'),
```

Comment:
Let's make this `label|default('Breadcrumb'|t)` so that if no custom label is passed in, it's still translateable. Or you could remove the default, since the Drupal template is passing in a label (that is translateable).

Reply:
I went with removing the default since Drupal is already passing in a translatable label.